### DISCUSSION TITLE
Error: P1010: User `johndoe` was denied access on the database `mydb.public`

### DISCUSSION THREAD

**Discussion by mbrsagor** [prisma/prisma#9331] [Q&A]
`npx prisma migrate dev --name "init" `
When I was run the command why raise the error?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
